Set DYLD_LIBRARY_PATH when building so that the library can be found even if we
use -DCMAKE_BUILD_WITH_INSTALL_RPATH:BOOL=ON.
https://savannah.nongnu.org/bugs/index.php?65948
--- doc/CMakeLists.txt.orig	2025-12-14 03:03:45.000000000 -0600
+++ doc/CMakeLists.txt	2025-12-14 03:19:21.000000000 -0600
@@ -17,7 +17,8 @@
   set(DOC_MAN ${CMAKE_CURRENT_BINARY_DIR}/simulavr.1.gz)
   
   add_custom_command(OUTPUT ${DOC_MAN}
-                     COMMAND ${HELP2MAN} -o ${CMAKE_CURRENT_BINARY_DIR}/simulavr.1 "$<TARGET_FILE:simulavr>"
+                     VERBATIM
+                     COMMAND ${CMAKE_COMMAND} -E env DYLD_LIBRARY_PATH=$<TARGET_FILE_DIR:sim> ${HELP2MAN} -o ${CMAKE_CURRENT_BINARY_DIR}/simulavr.1 "$<TARGET_FILE:simulavr>"
                      COMMAND ${GZIP} -n9 ${CMAKE_CURRENT_BINARY_DIR}/simulavr.1)
   add_custom_target(doc-man DEPENDS ${DOC_MAN})
   add_dependencies(progdoc doc-man)
